### PR TITLE
 Make Alert Dialog boxes Uniform #960

### PR DESCRIPTION
**Pull request by prajurock**
Make Alert Dialog boxes Uniform #960

Changes: Changes in Style

Screenshots/GIF for the change: [If possible, please add relevant screenshots/GIF]
whatsapp image 2019-02-12 at 8 29 27 am 3
whatsapp image 2019-02-12 at 8 29 27 am 2
whatsapp image 2019-02-12 at 8 29 27 am 1
whatsapp image 2019-02-12 at 8 29 27 am
whatsapp image 2019-02-12 at 8 29 27 am 5
whatsapp image 2019-02-12 at 8 29 27 am 4

### CODE REVIEW

**Comment by prajurock** [comment]
@soloturn @abdulwd @siddharth2010 I closes my previous PR due to some issue. Please review this

**Comment by prajurock** [comment]
@siddharth2010 Check now, I made the changes

**Comment by prajurock** [comment]
whatsapp image 2019-02-12 at 6 51 29 pm 2
whatsapp image 2019-02-12 at 6 51 29 pm 1
whatsapp image 2019-02-12 at 6 51 29 pm

**Comment by siddharth2010** [comment]
@prajurock , could you also add the screenshot of the language selection dialog in the dark mode?

**Comment by prajurock** [comment]
whatsapp image 2019-02-12 at 7 23 56 pm

**Comment by prajurock** [comment]
@siddharth2010  Done.

**Comment by siddharth2010** [comment]
Nice work @prajurock . However, the heights of the language dialog are not the same in day and night modes:
image

Also the selected radio button in night mode is green in color (not blue)

Also, the cancel button and the radio buttons on the language activity are white in night mode. Try to make them the same as the accent color (like in day mode).

See if you could fix that.

**Comment by abdulwd** [comment]
> @soloturn @abdulwd @siddharth2010 I closes my previous PR due to some issue. Please review this

@prajurock You can share your issue with us, multiple PRs for the same issue should not be created.

**Comment by prajurock** [comment]
> Nice work @prajurock . However, the heights of the language dialog are not the same in day and night modes:
> image
> 
> Also the selected radio button in night mode is green in color (not blue)
> 
> Also, the cancel button and the radio buttons on the language activity are white in night mode. Try to make them the same as the accent color (like in day mode).
> 
> See if you could fix that.

Ok, I will see. and let you know.

**Comment by soloturn** [comment]
@prajurock this is ok now?

**Comment by prajurock** [comment]
> @prajurock this is ok now?

No. I will do it by tomorrow.

**Comment by soloturn** [comment]
> > @prajurock this is ok now?
> 
> No. I will do it by tomorrow.

cool!

File: app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java [hunk 1]
```diff
@@ -962,6 +962,12 @@ public void openExternalUrl(Intent intent) {
   }
 
   private void externalLinkPopup(Intent intent) {
+    int warningResId;
+    if (sharedPreferenceUtil.nightMode()) {
+      warningResId = R.drawable.ic_warning_white;
```
Comment (abdulwd) [inline app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java]: Adding if-else everywhere in the code will unnecessarily complicate the code. You should move this to XML using this [answer](https://stackoverflow.com/a/13735919/7792129).

Comment (soloturn) [reply app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java]: interesting, @prajurock what you think about it?

**Comment by abdulwd** [comment]
Closing due to inactivity. Please reopen this if required.

**Pull request closed without merging**